workshop/exercise-5/README.md

```diff
@@ -1,85 +1,58 @@
-## Deploy vnext Version and Apply Traffic Shifting
+## Tag revisions to generate custom app URLs
 
-Did you notice that the Fibonacci sequence started with 1? Most would argue that the sequence should actually start with 0. There's a vnext version of the application at the vnext branch in the github project that starts the sequence with 0 instead of 1. This container image has been built and deployed to dockerhub, and tagged as vnext. We'll deploy that as v2 of our app.
-
-### Deploy vnext
-1. Let's deploy vnext, again using a docker image on dockerhub. Maybe we want to slowly roll users over from our old version to the new version, or do some A/B testing of the new version to see what users like better. Let's see what the yaml for this looks like.
+What if we wanted to create subroutes for each of the two revisions we created earlier? Maybe one of the revisions is for staging, and one is for production? Knative will automatically generate subroutes for us when we tag the Revisions in our Service.
```

Comment:
Instead of "subroutes", let's just use something like "version specific URLs" or something less technical like that.

Reply:
the use of the word 'tag' here could be misleading since giving the revision names in the previous exercises could be interpreted as  'tag'.  What about something like:
You can ask Knative to assign a URL to a specific revision by using the `--tag` option.

Instead of the "Knative will automatically.... " sentence.

Reply:
naming and tagging a revision are different, but i updated the wording here anyway.